### PR TITLE
Adds issues templates

### PR DESCRIPTION
**Pull request by lomky**
Finishes #1 
Finishes #2 

Integrates the templates with GitHub so that they will appear when we hit the New Issue button.

[See GitHub's docs for more infor](https://docs.github.com/en/github/building-a-strong-community/about-issue-and-pull-request-templates)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/task.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+name: Task
+about: Template for a new task
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Task
+<!-- Describe what needs to be done -->
```
Comment (lomky) [inline .github/ISSUE_TEMPLATE/task.md:11]: this syntax means the line won't show on the issue if you don't bother to delete it

File: .github/ISSUE_TEMPLATE/task.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+name: Task
+about: Template for a new task
+title: ''
+labels: ''
+assignees: ''
+
+---
```
Comment (lomky) [inline .github/ISSUE_TEMPLATE/task.md:8]: This section is metadata for github and won't show

File: .github/ISSUE_TEMPLATE/task.md (hunk 1 above)
Comment (kalvinwang) [reply .github/ISSUE_TEMPLATE/task.md:11]: oh nice!

**Review by kalvinwang — APPROVED** [review]
Thanks!

**Pull request merged**